substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -101,28 +101,28 @@ pub type Evidence<T, I> = BoundedVec<u8, <T as Config<I>>::EvidenceSize>;
 
 /// The status of the pallet instance.
 #[derive(Encode, Decode, Eq, PartialEq, Clone, TypeInfo, MaxEncodedLen, RuntimeDebug)]
-pub struct ParamsType<Balance, BlockNumber, const RANKS: usize> {
+pub struct ParamsType<Balance, BlockNumber> {
 	/// The amounts to be paid when a member of a given rank (-1) is active.
-	active_salary: [Balance; RANKS],
+	active_salary: Vec<Balance>,
```

Comment:
This changes the encoding of the type.

Reply:
It should probably be `BoundedVec<Balance, Ranks>` and `Ranks` passed in as `Get<u32>` generic.